docs/overview/title-transfer-example.md

```diff
@@ -0,0 +1,175 @@
+
+### Actions
+1. Alice:
+    - change holder
+    - endorse
+    - surrender document
+2. Bob:
+    - no access
+3. Charlie:
+    - no access
+
+We will do **endorse change of ownership** first.
+
+What this command does is it sets the states (holder and owner) of the token to a given address, in this case we will endorse the change of ownership to Charlie.
+
+```
+open-attestation title-escrow endorse-change-owner --token-registry <TOKEN_REGISTRY_ADDRESS> --tokenId <TOKEN_ID> --newOwner <CHARLIE_ADDRESS> --newHolder <CHARLIE_ADDRESS> -n ropsten --key <ALICE_PTE_KEY>
+```
+
+If this transaction is mined and successful, the state of the transferable record will be different.
+
+---
+## Change Holder
+
+The next action we will demonstrate  is **change holder**, but first, the present states and actions.
+
+### Present state:
+The token belongs to `Charlie` (both owner and holder are the address of Charlie)
+
+### Actions:
+1. Alice:
+    - no access
+2. Bob:
+    - no access
+3. Charlie:
+    - change holder
+    - endorse change of ownership
+    - surrender document
+
+Now we will do the **change holder** command.
+
+What this command does is it just sets the holder state to a new address.
+
+In this case we will set the holder state to `Bob`, owner state remains as `Charlie` address
+
+```bash
+open-attestation title-escrow change-holder --token-registry <TOKEN_REGISTRY_ADDRESS> --tokenId <TOKEN_ID> --to <TO> -n ropsten --key <CHARLIE_PTE_KEY>
+```
+
+Do take note that the private key supplied should be that of Charlie instead of Alice since Charlie currently holds and owns the token.
+
+If this command is successful, we will yet again advance the state of the token.
+
+---
+
+## Nominate Change of Ownership
+
+### Present state:
+The token is held by `Bob`, token is owned by `Charlie`
+
+### Actions:
+1. Alice:
+    - no access
+2. Bob:
+    - change holder
+3. Charlie:
+    - nominate change of ownership (approve new owner)
+
+Next, we will do **nominate change of ownership** as we have already demonstrated **change holder**.
+
+What this command does is to allow `Charlie`, the current owner, to suggest a new owner of the token. 
+
+In this case we will suggest `Alice` to be the new owner.
+
+Does this lead to any change in state (holder, owner)? The answer is no, not yet.
+
+```
+open-attestation title-escrow nominate-change-owner --token-registry <TOKEN_REGISTRY_ADDRESS> --tokenId <TOKEN_ID> --newOwner <NEW_OWNER_ADDRESS> -n ropsten --key <CHARLIE_PTE_KEY>
+```
+
+If this action is successful, then an additional action should be present on `Bob`.
+
+---
+
+## Endorse Transfer of Ownership
+
+### Present state:
+The token is held by `Bob`, token is owned by `Charlie`, no change in state from previous action.
+
+### Actions:
+1. Alice:
+    - no access
+2. Bob:
+    - change holder
+    - endorse transfer of ownership
+3. Charlie:
+    - nominate change of ownership (approve new owner)
```

Comment:
```suggestion
2. Bob:
    - change holder (approve new owner)
    - endorse transfer of ownership
3. Charlie:
    - nominate change of ownership
```

in a way, Bob decides whether to approve the new nominated owner by charlie. at this point charlie can still kinda suggest other ppl as new owner.

i thought the `(approve new owner)` text might misled, wdyt?

Reply:
yup you are right, in this case maybe we should just get rid of the `(approve new owner)` text